test/OrchardCore.Commerce.Tests.UI/Tests/OrderTests/BehaviorOrderTests.cs

```diff
@@ -0,0 +1,157 @@
+using Atata;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OrchardCore.Commerce.Tests.UI.Tests.OrderTests;
+
+public class BehaviorOrderTests : UITestBase
+{
+    public BehaviorOrderTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    [Theory, Chrome]
+    public Task OrderEditorShouldWorkProperly(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            async context =>
+            {
```

Comment:
I'd add 4 methods for Quantity, ProductSku, UnitPriceValue and UnitPriceCurrencyIsoCode to reduce all those duplicate selectors like:
```csharp
By ByQuantity(int index) => By.Id($"OrderPart.LineItems[{index}].Quantity");
```